### PR TITLE
Formats code samples and updates link text

### PR DESCRIPTION
**Pull request by joeliedtke**
Updates environment variables page with changes for kubeflow/website #1732.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1733)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview:
https://deploy-preview-1733--competent-brattain-de2d6d.netlify.com/docs/pipelines/sdk/enviroment_variables/

File: content/docs/pipelines/sdk/enviroment_variables.md [hunk 1]
```diff
@@ -21,26 +21,28 @@ Set up your environment:
 In this example, you pass an environment variable to a lightweight Python 
 component, which writes the variable's value to the log.
 
-[Learn more about](https://www.Kubeflow.org/docs/pipelines/sdk/lightweight-python-components/).
+[Learn more about lightweight Python components.](https://www.Kubeflow.org/docs/pipelines/sdk/lightweight-python-components/)
```
[the remark this answers is not in the document]
Comment (joeliedtke) [reply content/docs/pipelines/sdk/enviroment_variables.md]: Yikes, thanks for noticing the URLs... I updated the other links in this doc to use relative links too.

PTAL

**Comment by sarahmaddox** [comment]
Thanks @joeliedtke!
/lgtm
/approve

**Pull request merged**